starknet_py/tests/e2e/account/account_test.py

```diff
@@ -320,6 +322,11 @@ async def test_deploy_account_uses_custom_calldata(
         deployer_address=0,
     )
 
+    res = await fee_contract.functions["transfer"].invoke(
+        recipient=address, amount=int(1e25), max_fee=MAX_FEE
+    )
+    await res.wait_for_acceptance()
```

Comment:
Shouldn't the `deploy_account_details_factory` also be prefunding the generated address? I though that it was the case.

Reply:
It seems, the factory is used here purely for random `key_pair` and `salt`, as `address` is ignored and `calldata` is overridden. Hence the need for the transfer to a new address.